torch/lib/c10d/test/ProcessGroupGlooTest.cpp

```diff
@@ -381,6 +383,7 @@ int main(int argc, char** argv) {
     }
   }
 
+  // test SIGKILL
```

Comment:
Should this be a separate test case?

Reply:
I will move this in the 3rd PR in this stack.